### PR TITLE
Wahlplattform VOTE Hotfix 3

### PR DESCRIPTION
**Pull request by garamond**
image

### CODE REVIEW

File: components/Vote/ElectionDiscussionPage.js [hunk 1]
```diff
@@ -113,18 +121,36 @@ const query = gql`
     id
     discussion {
       id
+      comments {
+        id
+        nodes {
```
Comment (tpreusse) [inline components/Vote/ElectionDiscussionPage.js]: query `totalCount` instead of nodes

File: components/Vote/ElectionDiscussionPage.js [hunk 2]
```diff
@@ -113,18 +121,36 @@ const query = gql`
...
+      }
     }
    }
   ${ELECTION_COOP_MEMBERS_SLUG}: election(slug: "${ELECTION_COOP_MEMBERS_SLUG}") {
     id
     discussion {
       id
+      comments {
+        id
+        nodes {
```
Comment (tpreusse) [inline components/Vote/ElectionDiscussionPage.js]: query `totalCount` instead of nodes

File: components/Vote/ElectionDiscussionPage.js [hunk 3]
```diff
@@ -113,18 +121,36 @@ const query = gql`
...
+      }
     }
    }
   ${VOTING_COOP_BOARD_SLUG}: voting(slug: "${VOTING_COOP_BOARD_SLUG}") {
     id
     discussion {
       id
+      comments {
+        id
+        nodes {
```
Comment (tpreusse) [inline components/Vote/ElectionDiscussionPage.js]: query `totalCount` instead of nodes

File: components/Vote/ElectionDiscussionPage.js [hunk 4]
```diff
@@ -87,6 +93,8 @@ const DiscussionPage = ({ router, data, vt }) => {
                             )
                             }
                           </Link>
+
+                          <span {...styles.count}><Icon size={17} fill={colors.primary} /> {data[id] && data[id].discussion.comments.nodes.length}</span>
```
Comment (tpreusse) [inline components/Vote/ElectionDiscussionPage.js]: use `totalCount`

File: components/Vote/ElectionDiscussionPage.js (hunk 1 above)
Comment (garamond) [reply components/Vote/ElectionDiscussionPage.js]: 👍

**Pull request merged**